pdns/dbdnsseckeeper.cc

```diff
@@ -91,7 +91,11 @@ int DNSSECKeeper::addKey(const DNSName& name, bool setSEPBit, int algorithm, int
   }
   DNSSECPrivateKey dspk;
   shared_ptr<DNSCryptoKeyEngine> dpk(DNSCryptoKeyEngine::make(algorithm));
-  dpk->create(bits);
+  try{
+    dpk->create(bits);
+  } catch (std::runtime_error error){
+    throw runtime_error("Wrong bit size!");
```

Comment:
please use a more specific exception, and i think the error message could be a lot clearer (`Unsupported key size`?)